### PR TITLE
Joss paper [in prep]

### PR DESCRIPTION
**Pull request by AndrewAnnex**
PR to track progress on Paper for JOSS.org

Not all co-authors have replied to be included, so some contributors are not listed in the paper.md file (but are in the codemeta.json file)

Currently the author ordering is alphabetical after me (well either way I'm generally first 😄 ) as I did not want to order by number of contributions or other metrics. 

If you have edits please comment on the lines and I'll work to integrate them in. 
IIRC joss needs the individual author contributions on the paper.md file which I will integrate in as we go.  

~~for those I have not heard back from ([☑️]@michaelaye []@drjmcauliffe [☑️]@MStefko [☑️]@jdiazdelrio  [☑️]@helgee [☑️]@murashin ) please let me know if you do or do not wish to be included (if so please provide full name, affiliation, and ORCID)~~
### update:

Just waiting to hear back from @drjmcauliffe

### CODE REVIEW

**Comment by helgee** [comment]
Full name: Helge Eichhorn
Affiliation: Planetary Transportation Systems GmbH
ORCID: https://orcid.org/0000-0003-0303-5199

Thanks! Co-authorship for a single commit seems like a deal that is heavily skewed in my favour but I will gladly take it 😊

**Comment by MStefko** [comment]
Full name: Marcel Stefko
Affiliation: ETH Zurich
ORCID: https://orcid.org/0000-0002-7736-2611

I gladly accept the offer, thanks both for this and managing this amazing piece of software!

**Comment by murashin** [comment]
Full name: Shin-ya Murakami
Affiliation: GFD Dennou Club
ORCID: https://orcid.org/0000-0002-7137-4849

Thank you for the offer. I think my contribution is very small, but I gladly accept it.

File: joss/codemeta.json [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      "email": "",
+      "name": "Shin-ya Murakami",
+      "affiliation": "GFD Dennou Club"
+    },
+    {
+      "@id": "https://orcid.org/0000-0003-0303-5199",
+      "@type": "Person",
+      "email": "",
+      "name": "Helge Eichhorn",
+      "affiliation": "Planetary Transportation Systems, GmbH"
```
Comment (helgee) [inline joss/codemeta.json]:
```suggestion
      "affiliation": "Planetary Transportation Systems GmbH, Berlin, Germany"
```

File: joss/paper.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+affiliations:
+ - name: Johns Hopkins University, Baltimore, MD 21218, USA.
+   index: 1
+ - name: General Public.
+   index: 2
+ - name: Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA 91109, USA.
+   index: 3
+ - name: Latchmoor Services, LLC.
+   index: 4
+ - name: Planetary Transportation Systems, GmbH.
```
Comment (helgee) [inline joss/paper.md]:
```suggestion
 - name: Planetary Transportation Systems GmbH, Berlin, Germany.
```

**Comment by drjmcauliffe** [comment]
Full Name: Jonathan Mc Auliffe
Affiliation: DLR Gesellschaft für Raumfahrtanwendungen (GfR) mbH

Sorry for the late feedback and thanks for the inclusion. As said by other above, my minor contributions barely merit inclusion... but I'll take it. As always, great work on this software.

**Pull request merged**